main.go

```diff
@@ -118,15 +118,26 @@ func destroyDNS(c *conf, hc *http.Client) error {
 	req.Header.Add("Accept", "application/json")
 	req.Header.Add("Authorization", fmt.Sprintf("Basic %s", c.konsAPIKey))
 
-	_, err = hc.Do(req)
+	response, err := hc.Do(req)
 	if err != nil {
 		return err
 	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		// if status is not 200, log it, but do not consider it as a service failure
```

Comment:
Yeah, I'm aware of that - there are several unhappy use cases actually which need to be handled differently (e.g. there were no DNS record prior of destroying it, failing to delete existing record, etc(?) ). I will think about a consistent solution & create a separate branch for that